site/content/docs/5.0/examples/starter-template/index.html

```diff
@@ -5,46 +5,46 @@
   - "starter-template.css"
 ---
 
-<nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
-  <div class="container-fluid">
-    <a class="navbar-brand" href="#">Navbar</a>
-    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
+<div class="col-lg-8 mx-auto p-3 py-md-5">
+  <header class="d-flex align-items-center pb-3 mb-5 border-bottom">
```

Comment:
need to check logo

Reply:
> * some sample need brand alignement. Maybe fix the design on accordions and create other PR for Orange branded samples
> * no need to add dist files in this MR only for release, not sure if this PR is intended to be the release or just the bootstrap merge
> * What's that new dist/css/boosted-rtl.rtl.css.map? Is there a merge side effect that produce new files?
> * before release need to change the boosted version in _config.yml but not at time (if this PR is not for release)

It's meant to be the release of Boosted beta3

Reply:
will be adressed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/380